cdap-common/src/main/java/io/cdap/cdap/internal/app/store/RunRecordDetail.java

```diff
@@ -203,18 +203,18 @@ public static Builder builder(RunRecordDetail record) {
    * Builds RunRecordMetas.
    */
   public static class Builder extends RunRecord.Builder<Builder> {
```

Comment:
You can parameterize if with <T extends Builder> and pass it to the parent. This would allow you not to overide all setters in RunRecordDetailWithExistingStatus.Builded

Reply:
done